templates/pbiviz.json.template

```diff
@@ -18,6 +18,7 @@
         "icon": "assets/icon.png"
     },
     "externalJS": [
+        "node_modules/lodash/lodash.min.js",
```

Comment:
I think the dependency should be in dataviewutils and not in powerbi-visual-tools - no? if you jump to version 1.1.1 then it should install lodash i assume.